graylog2-server/src/main/java/org/graylog/security/certutil/ca/CACreator.java

```diff
@@ -59,6 +62,15 @@ public KeyStore createCA(final char[] password,
         } catch (Exception e) {
             throw new CACreationException("Failed to create a Certificate Authority", e);
         }
+    }
+
+    public KeyStore uploadCA(KeyStore keyStore, final char[] password, String cert) throws CACreationException {
```

Comment:
Is `keystore` parameter needed? You seem to invoke this method with `null` as a first parameter.
Don't you face problems we have had before, which made you introduce explicit Base64 encoding?
Is it maybe reasonable to repeat the approach from `KeystoreMongoStorage` here and avoid `aws` dependency?
```
            try (ByteArrayInputStream bais = new ByteArrayInputStream(Base64.getDecoder().decode(keystoreAsString.get()))) {
                KeyStore keyStore = KeyStore.getInstance(PKCS12);
                keyStore.load(bais, password);
                return Optional.of(keyStore);
            }
```